app/main.py

```diff
@@ -1,2 +1,56 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    load_factor = 2 / 3
+
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.length = 0
+        self.threshold = int(self.capacity * self.load_factor)
+        self.hash_table = self.create_hash_table()
+
+    def create_hash_table(self) -> list:
+        return [[] for _ in range(self.capacity)]
+
+    def resize(self) -> None:
+        hash_table = self.hash_table
+        self.length = 0
+        self.capacity *= 2
+        self.threshold = int(self.capacity * self.load_factor)
+        self.hash_table = self.create_hash_table()
+
+        for element in hash_table:
+            if element:
+                self.__setitem__(element[0], element[2])
+
+    def __getitem__(self, key: Hashable) -> Any:
+        hash_key = hash(key)
+        hash_index = hash_key % self.capacity
+
+        while self.hash_table[hash_index]:
+            if (self.hash_table[hash_index][0] == key
+                    and self.hash_table[hash_index][1] == hash_key):
```

Comment:
There is no need to compare hash_keys if the keys are equal
It would be the same without 'and' part